mzLib/FlashLFQ/FlashLfqEngine.cs

```diff
@@ -236,6 +259,93 @@ public FlashLfqResults Run()
                 _peakIndexingEngine.ClearIndex();
             }
 
+
+            //IsoTracker
+            if (IsoTracker)
+            {
+                IsoTrackerIsRunning = true; // Turn on the flag, then we will use the separate indexEngine for each files
+                IsobaricPeakInDifferentRun = new Dictionary<string, Dictionary<Tuple<int, double, double>, List<ChromatographicPeak>>>();
+                var idGroupedBySeq = _allIdentifications.Where(p=> p.BaseSequence != p.ModifiedSequence).GroupBy(p => p.ModifiedSequence);
+                // We only consider the modified sequence in the IsoTracker
```

Comment:
You should group by base sequence and mass. e.g. PEP[Oxidation]TIDEK and PEPTI[Oxidation]DEK are isobaric and should be in the same group